### DISCUSSION TITLE
Add label to prompt from Langfuse UI

### DISCUSSION THREAD

**Discussion by charlieviettq** [langfuse/langfuse#2366] [Support] [answered]
### Describe the feature or potential improvement

Can we have feature for adding label for prompt from `Langfuse UI`. It will be easy to retrieve prompt version with a label for Prompt A/B Testing without create 2 prompts, only need to create 2 version with specific label.

**Comment by marcklingen** [comment, marked as answer]
You can add a label to a prompt from the ui:

https://github.com/langfuse/langfuse/assets/2834609/e6d79260-774a-48d7-88a1-d8de856605a3

You can use this for a/b testing by using two labels, e.g. `prod-a` and `prod-b`. In cases where you do not want to run an a/b test you could just apply both labels to the same version.

**Reply by charlieviettq** [reply]
Nice, thanks for your guides!
